api/src/main/resources/liquibase.xml

```diff
@@ -8,5 +8,27 @@
         See http://www.liquibase.org/manual/home#available_database_refactorings
         for a list of supported elements and attributes
     -->
+
+    <changeSet id="2018Jun28-1" author="Chew Chia Shao Yuan">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT COUNT(*) FROM
+                patient_identifier_type WHERE uuid ='a5d38e09-efcb-4d91-a526-50ce1ba5011a';
+            </sqlCheck>
+        </preConditions>
+        <comment></comment>
```

Comment:
Missing comment

Reply:
added "Creates a Patient Identifier Type which acts as the MPI Global Identifier for the Demo of the MPI Functionality with OpenEMPI"